helloapache/README.md

```diff
@@ -1,73 +1,57 @@
-# helloapache
+# Hello Apache
 
-This is an atomic application based on the nulecule specification. Kubernetes and native docker are currently the only supported providers. You'll need to run this from a workstation that has the atomic command.  If you wish to use the kubernetes provider, you will also need a kubectl client that can connect to a kubernetes master.
+### Description
 
-It's a single container application based on the centos/httpd image, but you can use your own.
+A helloworld example using the centos/apache container.
 
-## Option 1: Non-interactive defaults
+### Deployment
 
-Run the image. It will automatically use kubernetes as the orchestration provider.
+__Attended deployment__
```

Comment:
I think we need to describe this in a different way than "attended" vs "unattended" - I'm confused about what this really means. I see the difference based on the content. but I don't get this from "attended".